src/main/java/org/cbioportal/persistence/StudyViewRepository.java

```diff
@@ -29,6 +29,8 @@ public interface StudyViewRepository {
     
     List<AlterationCountByGene> getMutatedGenes(StudyViewFilterContext studyViewFilterContext);
     
+    List<AlterationCountByGene> getMutatedGenes(StudyViewFilterContext studyViewFilterContext, List<String> hugoGeneSymbols);
```

Comment:
```suggestion
    Map<String, Integer> getTotalProfiledCounts(StudyViewFilterContext studyViewFilterContext, String alterationType, String... genesToQuery);
```

This is what I want. So when calling the function you can do the following...

```java
        studyViewRepository.getTotalProfiledCounts(studyViewFilter, alterationType, "TP53", "GENEA");
        //or
studyViewRepository.getTotalProfiledCountsByAlterationType(studyViewFilterContext, alterationType.toString());
```

Reply:
Oh wow that's cool